src/query/sql/src/planner/binder/util.rs

```diff
@@ -86,4 +91,131 @@ impl Binder {
         }
         Ok(())
     }
+
+    pub fn fully_table_identifier<'b>(
+        &'b self,
+        catalog: &Option<Identifier>,
+        database: &Option<Identifier>,
+        table: &'b Identifier,
+    ) -> FullyTableIdentifier<'_> {
+        let Binder {
+            ctx,
+            name_resolution_ctx,
+            dialect,
+            ..
+        } = self;
+        let catalog = catalog.to_owned().unwrap_or(Identifier {
+            span: None,
+            name: ctx.get_current_catalog(),
+            quote: Some(dialect.default_ident_quote()),
+            is_hole: false,
+        });
+        let database = database.to_owned().unwrap_or(Identifier {
+            span: None,
+            name: ctx.get_current_database(),
+            quote: Some(dialect.default_ident_quote()),
+            is_hole: false,
+        });
+        FullyTableIdentifier {
+            name_resolution_ctx,
+            dialect: *dialect,
```

Comment:
Mysql handler not correctly set mysql dialect.
Should we force mysql dialect?
#5372

Reply:
Please review this comment. @andylokandy But I think force set is not a good idea.

Reply:
I think keeping now is better.